### ISSUE TITLE
Reconsider having the total number of nodes metrics back

### ISSUE THREAD

**Issue by diegoakechi** [ClusterLabs/ha_cluster_exporter#125]
The change introduced on https://github.com/ClusterLabs/ha_cluster_exporter/pull/123 have removed the total number of pacemaker configured nodes since it can be obtained via `count()`. 

We should review this decision since the current cluster nodes metrics involves the list of the cluster nodes status too, what makes one node appear more than once, and what could be confusing to the user. See:

```
ha_cluster_pacemaker_nodes{node="vmdrbd01",status="dc",type="member"} 1
ha_cluster_pacemaker_nodes{node="vmdrbd01",status="expected_up",type="member"} 1
ha_cluster_pacemaker_nodes{node="vmdrbd01",status="online",type="member"} 1
ha_cluster_pacemaker_nodes{node="vmdrbd02",status="expected_up",type="member"} 1
ha_cluster_pacemaker_nodes{node="vmdrbd02",status="online",type="member"} 1
```

In that sense, I would consider to bring the total nodes configured metric back, since the many possible status duplication makes it very error prone to do the proper count.

**Comment by stefanotorresi** [comment]
@diegoakechi I'm not sure about this.

We used the naming as the main deal-breaker with the `*_total` metrics: in Prometheus lingo, anything that is named `something_total` is an accumulator; so it's not a Gauge, it's a Counter, and as such, it can only increase over time.

But this was only the straw that broke the camel’s back: it made us do the leap and remove but it wasn't the only reason; after all, if we really wanted, we could have probably found a new naming that worked for both Prometheus conventions and us.

No, it wasn't just about naming; there is a more fundamental problem with `ha_cluster_pacemaker_nodes` specifically, and also all the metrics that have a composite cardinality.

Since we added the `status` label to the cardinality, it is not clear what a generic "count" metric would be about; what would `ha_cluster_pacemaker_nodes_count` mean? Is it the count of online nodes? Of all the nodes? Does it include the offline ones? What about the DC?
The point is, you would need to still take the label into account, and at that point you might as well express this explicitly in a query like `count(ha_cluster_pacemaker_nodes{status="online"})`.
